crates/meilisearch-types/src/keys.rs

```diff
@@ -144,6 +144,21 @@ impl Key {
         }
     }
 
+    pub fn default_management() -> Self {
+        let now = OffsetDateTime::now_utc();
+        let uid = Uuid::new_v4();
+        Self {
+            name: Some("Default Read-Only Admin API Key".to_string()),
+            description: Some("Use it to peek into the instance in a read-only mode.".to_string()),
+            uid,
+            actions: vec![Action::AllGet, Action::KeysGet],
+            indexes: vec![IndexUidPattern::all()],
+            expires_at: None,
+            created_at: now,
+            updated_at: now,
+        }
+    }
```

Comment:
Huum, since I think we should name the key the same way internally and externally with the `Read-Only admin` term.

Also, the description should be clearer on the fact that you should not share this key more than the original admin key.

I wrote a small sentence, but @guimachiavelli, you may be interested in reviewing the description here.
Iirc it's exposed on the cloud as well with its description, so people are going to see it even if they don't read the documentation at all.

```suggestion
    pub fn default_read_only_admin_key() -> Self {
        let now = OffsetDateTime::now_utc();
        let uid = Uuid::new_v4();
        Self {
            name: Some("Default Read-Only Admin API Key".to_string()),
            description: Some("Use it to peek into the instance in a read-only mode. Caution: This key gives you access to all the other api keys. Do not expose it on a public frontend.".to_string()),
            uid,
            actions: vec![Action::AllGet, Action::KeysGet],
            indexes: vec![IndexUidPattern::all()],
            expires_at: None,
            created_at: now,
            updated_at: now,
        }
    }
```